Keas.Mvc/Services/SecurityService.cs

```diff
@@ -244,6 +246,26 @@ public async Task<List<Role>> GetUserRolesInTeamOrAdmin(Team team)
             return userPermissions;
         }
 
+        public async Task<List<Role>> GetUserRolesInTeamOrAdmin(string teamSlug)
+        {
+            var userId = _contextAccessor.HttpContext.User.Identity.Name;
+
+            var userPermissions = await _dbContext.TeamPermissions
+                .Where(x => x.Team.Slug == teamSlug && x.User.Id == userId)
+                .Select(tp => tp.Role)
+                .AsNoTracking()
+                .ToListAsync();
+
+            var admin = await _dbContext.SystemPermissions
```

Comment:
@jcubbage This is good, but also we should discuss if Admins should be able to see this info or not.  I would argue a system admin should not be able to see a team's expiring keys, for example.  An admin should be able to emulate, create teams, and probably add themselves to someone else's team.  But I think it would simplify so much if we stopped checking for admin permissions

Reply:
That makes sense....but I'd say make a new issue and do that in a new branch since so much of the app will get updated. Maybe make only the TeamAdmin page include Admins (so they can ass uses to the team). That way, if you want to get to assets or reports, you need to emulate. Restrict the policy to be team permissions only, and decorate only desired actions with admin.

Reply:
Yea I agree, I'll make a new issue for later